### PR TITLE
Add GhostNet into hubs (CVPR 2020)

### PR DESCRIPTION
**Pull request by iamhankai**
The GhostNet architecture is based on an Ghost module structure which generate more features from cheap operations. Based on a set of intrinsic feature maps, a series of cheap operations are applied to generate many ghost feature maps that could fully reveal information underlying intrinsic features. Experiments conducted on benchmarks demonstrate that the superiority of GhostNet in terms of speed and accuracy tradeoff.

| Model structure | FLOPs       | Top-1 acc   | Top-5 acc   |
| --------------- | ----------- | ----------- | ----------- |
|  GhostNet 1.0x  | 142M        | 73.98       | 91.46       |

You can read the full paper at this [link](https://arxiv.org/abs/1911.11907) and more details at https://github.com/huawei-noah/ghostnet/tree/master/pytorch.

Comparison with other small CNNs.

### CODE REVIEW

File: pytorch_vision_ghostnet.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+```
+
+### Model Description
+
+The GhostNet architecture is based on an Ghost module structure which generate more features from cheap operations. Based on a set of intrinsic feature maps, a series of cheap operations are applied to generate many ghost feature maps that could fully reveal information underlying intrinsic features. Experiments conducted on benchmarks demonstrate that the superiority of GhostNet in terms of speed and accuracy tradeoff.
+
+| Model structure | FLOPs       | Top-1 acc   | Top-5 acc   |
+| --------------- | ----------- | ----------- | ----------- |
+|  GhostNet 1.0x  | 142M        | 73.98       | 91.46       |
```
Comment (ailzhang) [inline pytorch_vision_ghostnet.md:74]: Will be great to mention briefly here this accuracies are for which dataset/benchmark.

Comment (iamhankai) [reply pytorch_vision_ghostnet.md:76]: updated: https://github.com/pytorch/hub/pull/98/commits/e9fd124c32d2e21e6ab35152a8d7643fdec33b0d

**Comment by ailzhang** [comment]
@iamhankai The CI failure is also real I think

**Comment by iamhankai** [comment]
> @iamhankai The CI failure is also real I think

Passed.

**Comment by soumith** [comment]
thanks a lot for your contribution!

**Pull request merged**